trino-aws-proxy/src/test/java/io/trino/aws/proxy/server/credentials/http/TestHttpCredentialsProvider.java

```diff
@@ -57,8 +66,16 @@ public static class Filter
         public TestingTrinoAwsProxyServer.Builder filter(TestingTrinoAwsProxyServer.Builder builder)
         {
             TestingHttpServer httpCredentialsServer;
+            Map<String, String> expectedHeaders = ImmutableMap.<String, String>builder()
```

Comment:
nit: could we use `ImmutableMap.of(...)`? Its less verbose

Reply:
I usually tend to prefer using the builder if there are more than 1 or 2 entries. The `.of(...)` syntax gets a bit messy IMHO above that and you end up relying on formatting to keep each key & its value on the same line, it otherwise is quite unreadable.

No strong opinion, happy to change it if you prefer!